editoast/src/main.rs

```diff
@@ -164,3 +171,55 @@ fn generate(
     }
     Ok(())
 }
+
+#[derive(Debug)]
+struct InfraNameAlreadyExistsError;
+
+impl Error for InfraNameAlreadyExistsError {}
+
+impl Display for InfraNameAlreadyExistsError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Infra name already exists")
+    }
+}
+
+fn import_railjson(
+    args: ImportArgs,
+    pg_config: PostgresConfig,
+    chartos_config: ChartosConfig,
+)    -> Result<(), Box<dyn Error + Send + Sync>> {
+    let conn = PgConnection::establish(&pg_config.url()).expect("Error while connecting DB");
+
+    // check if path is valid
+    let railjson_file_result = File::open(args.railjson_path);
+    let railjson_file = match railjson_file_result {
+        Ok(file) => file,
+        Err(error) => panic!("Problem opening the file: {:?}", error),
```

Comment:
If in case of error you `panic` then you can use the [expect](https://doc.rust-lang.org/std/result/enum.Result.html#method.expect) function.